src/libmongoc/src/mongoc/mongoc-uri.c

```diff
@@ -610,6 +610,67 @@ mongoc_uri_parse_auth_mechanism_properties (mongoc_uri_t *uri, const char *str)
    return true;
 }
 
+
+static bool
+mongoc_uri_check_srv_service_name (mongoc_uri_t *uri, const char *str)
+{
+   /* 63 character DNS query limit, excluding prepended underscore. */
+   const size_t mongoc_srv_service_name_max = 62;
```

Comment:
Is there any reason you're using the `u` suffix (unsigned) on literals below but not here?

Reply:
In an attempt to keep sign consistency. It is not as important in initializers and comparison expressions (particularly when the value is clearly within range of representable values), but lack-of may result in unexpected well-defined overflow (or lack thereof) in arithmetic expressions. However, for full/proper consistency, I should indeed be using the `u` suffix here as well.